idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1484,7 +1484,7 @@ struct BatchCancelRequest{
 
 struct StartBatchJobRequest{
     10: optional string domain
-    20: optional string jobID
+    20: optional SearchAttributes searchAttributes
```

Comment:
why do you need `SearchAttributes` while starting batch job? Also its odd to have it as 2nd parameter. If we are anyway breaking contract move it down..

Reply:
With this service, customers can start many batch jobs and the only way to manage(find) them is to put some tags. 
@mfateev  I can remove it if you both think it is unnecessary for now. We can definitely do it later when needed.